### ISSUE TITLE
should report bad CL options and exit

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1599]
cq-id: VisIt00004378
cq-submitter: Mark Miller
cq-submit-date: 01/27/04

I have inadvertently given visit bad command-line arguments on many occasions. However, it neither reports this to me nor fails to start because of them. Maybe I want to collect some debug logs and pass '-debg 5'. Only after I have run visit do I realize I've spelled it wrong.

Visit should check all command-line arguments and fail to start of it doesn't recognize any.

MCM -- 25Jan06 -- Scoring Rationale
-----------------------------------
I rated seriousness as "crash/wrong results" because prior to correcting a specific issue with '-timeout' CL option to engine, the viewer could indeed crash as a result of a missing arg. I haven't tested possibility of this happening for other CL options.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 06/21/2010 07:09 pm
Original update: 07/06/2010 09:18 pm
Ticket number: 12
